cli/chaosli/cmd/context.go

```diff
@@ -0,0 +1,388 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021 Datadog, Inc.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"math"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/DataDog/chaos-controller/types"
+	"github.com/briandowns/spinner"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+)
+
+const MAXTARGETSHOW = 10
+
+func contextTargetsSize(disruption v1beta1.Disruption) ([]string, error) {
+	spec := disruption.Spec
+	labels := spec.Selector.String()
+	level := spec.Level
+	podNamespaces := disruption.ObjectMeta.Namespace
+	rowNames := 1
+
+	if level == types.DisruptionLevelNode {
+		rowNames = 1
+	}
+
+	fmt.Println("Let's look at your targets...")
+
+	s := spinner.New(spinner.CharSets[38], 100*time.Millisecond)
+	s.Start()
+
+	size, err := getTargetSize(disruption)
+
+	if err != nil {
+		return nil, err
+	}
+
+	// Remove header NAME from consideration
+	size--
+	// If the size is 0, first check if changing the level will do anything, otherwise
+	// mention to the user that the labels they are using won't target anything
+
+	if size <= 0 {
+		errorString := fmt.Sprintf("\nThe label selectors chosen (%s) result in 0 targets, meaning this disruption would do nothing given the namespace/cluster/label combination.", labels)
+		newLevel := ""
```

Comment:
`newLevel` never gets set to anything other than this empty string

Reply:
yes another remnant of when I didnt have the namespace value on hand and the node and pod names were on different rows.